nifi-extension-bundles/nifi-workday-bundle/nifi-workday-processors/src/main/java/org/apache/nifi/processors/workday/GetWorkdayReport.java

```diff
@@ -90,77 +93,111 @@ public class GetWorkdayReport extends AbstractProcessor {
     protected static final String GET_WORKDAY_REPORT_JAVA_EXCEPTION_MESSAGE = "getworkdayreport.java.exception.message";
     protected static final String RECORD_COUNT = "record.count";
     protected static final String BASIC_PREFIX = "Basic ";
+    protected static final String BEARER_PREFIX = "Bearer ";
     protected static final String HEADER_AUTHORIZATION = "Authorization";
     protected static final String HEADER_CONTENT_TYPE = "Content-Type";
     protected static final String USERNAME_PASSWORD_SEPARATOR = ":";
 
     protected static final PropertyDescriptor REPORT_URL = new PropertyDescriptor.Builder()
-        .name("Workday Report URL")
-        .displayName("Workday Report URL")
-        .description("HTTP remote URL of Workday report including a scheme of http or https, as well as a hostname or IP address with optional port and path elements.")
-        .required(true)
-        .expressionLanguageSupported(FLOWFILE_ATTRIBUTES)
-        .addValidator(URL_VALIDATOR)
-        .build();
+            .name("Workday Report URL")
+            .displayName("Workday Report URL")
+            .description("HTTP remote URL of Workday report including a scheme of http or https, as well as a hostname or IP address with optional port and path elements.")
+            .required(true)
+            .expressionLanguageSupported(FLOWFILE_ATTRIBUTES)
+            .addValidator(URL_VALIDATOR)
+            .build();
+
+    public static AllowableValue BASIC_AUTH_TYPE = new AllowableValue(
+            "basic_auth",
```

Comment:
The `value` field should follow enum conventions, so all capitals with `BASIC_AUTH` would be best in this scenario.